libseccomp/tests/tests.rs

```diff
@@ -170,18 +173,33 @@ fn test_syscall_eq_i32() {
 }
 
 #[test]
+#[allow(deprecated)]
 fn test_get_syscall_name_from_arch() {
```

Comment:
```diff
@@ -175,21 +175,19 @@ fn test_syscall_eq_i32() {
 #[test]
 #[allow(deprecated)]
 fn test_get_syscall_name_from_arch() {
-    println!(
-        "test_get_syscall_from_name: Got syscall name of 5 on native arch as {}",
-        ScmpSyscall::from(5).get_name().unwrap(),
+    assert_eq!(
+        "openat",
+        ScmpSyscall::from(56)
+            .get_name_by_arch(ScmpArch::Aarch64)
+            .unwrap()
     );
-
-    println!(
-        "test_get_syscall_from_name: Got syscall name of 5 on ARM arch as {}",
-        ScmpSyscall::from(5)
-            .get_name_by_arch(ScmpArch::Arm)
-            .unwrap(),
-    );
-
     assert!(ScmpSyscall::from(10_000).get_name().is_err());
     assert!(ScmpSyscall::from(-1).get_name().is_err());

+    assert_eq!(
+        get_syscall_name_from_arch(ScmpArch::Native, 5).unwrap(),
+        ScmpSyscall::from(5).get_name().unwrap(),
+    );
     assert_eq!(
         get_syscall_name_from_arch(ScmpArch::Aarch64, 5).unwrap(),
         ScmpSyscall::from(5)
@@ -201,19 +199,13 @@ fn test_get_syscall_name_from_arch() {
 #[test]
 #[allow(deprecated)]
 fn test_get_syscall_from_name() {
-    println!(
-        "test_get_syscall_from_name: Got syscall number of open on native arch as {}",
-        ScmpSyscall::from_name("open").unwrap()
+    assert_eq!(
+        ScmpSyscall::from_name_by_arch("openat", ScmpArch::Aarch64).unwrap(),
+        56
     );
-
-    println!(
-        "test_get_syscall_from_name: Got syscall number of open on ARM arch as {}",
-        ScmpSyscall::from_name_by_arch("open", ScmpArch::Arm).unwrap()
-    );
-
-    println!(
-        "test_get_syscall_from_name: Got rewritten syscall number of socket on X32 arch as {}",
-        ScmpSyscall::from_name_by_arch_rewrite("socket", ScmpArch::X32).unwrap(),
+    assert_eq!(
+        ScmpSyscall::from_name_by_arch_rewrite("socket", ScmpArch::X86).unwrap(),
+        102
     );

     assert!(ScmpSyscall::from_name("no_syscall").is_err());
```

Reply:
Could you fix the testsas above?
1. `from_name_by_arch_rewrite` works on 32bit x86 (`ScmpArch::X86`).
2. I'd like to unify the architecture and the syscall which are tested.
3. Remove unecessary `println!`. Ref: https://github.com/libseccomp-rs/libseccomp-rs/issues/120